aten/src/ATen/native/Convolution.cpp

```diff
@@ -583,7 +597,7 @@ at::Tensor _convolution(
 
   check_shape_forward(input, weight_sizes, bias, params);
 
-  if (input.size(0) == 0) {    
+  if (input.size(0) == 0) {
```

Comment:
Trailing whitespaces.

Reply:
Thanks will fix.

Reply:
OK, actually seems I removed (not added) a trailing space. :)  This is a setting on my editor to removing trailing spaces on save.